tests/e2e/utils/utils.ts

```diff
@@ -27,26 +26,13 @@ export const generateTestProject = async (
 		throw new Error(`error: ${error}`);
 	}
 
-	await exec('npm init -y', { cwd: targetDir, encoding: 'utf-8' });
+	await exec(`touch package.json`, { cwd: `${targetDir}`, encoding: `utf-8` });
+	await exec('npm init -y', { cwd: `${targetDir}`, encoding: 'utf-8' });
```

Comment:
I am wondering if we still need to run npm init or not. I guess Michael made it implicit or I am wrong?